code/game/objects/structures/manned_turret.dm

```diff
@@ -47,10 +47,10 @@
 	if(user.incapacitated() || !istype(user))
 		return
 	if(do_after(user, 30, target = M))
+		M.forceMove(get_turf(src))
 		. = ..()
 		if(!.)
 			return
-		M.forceMove(get_turf(src))

```

Comment:
This should have stayed as-is, you never know why someone was failed to be buckled, what if they were buckled to a chair? They'd be forceMove'd out of it